Rakefile

```diff
@@ -81,10 +81,14 @@ namespace :gen do
     end
   end
 
-  desc "Resolve the standard in arch/, and write it to gen/resolved_arch/_"
-  task "resolved_arch" do
-    $resolver.cfg_arch_for("_")
-  end
+desc "Resolve the configuration CFG in arch/, and write it to gen/resolved_arch/<CFG>. Default CFG is the standard, \"_\"."
+   task "resolved_arch" do
+     cfg = ENV["CFG"]
+     if cfg.nil?
+       cfg = "_"
+     end
+     $resolver.cfg_arch_for(cfg)
+   end
```

Comment:
this is a minor nit... could you change the indentation to 2 spaces for these 7 lines to match the previous code and the project conventions? Thanks!